.github/workflows/fuzz_test.yml

```diff
@@ -0,0 +1,84 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+name: Fuzz Test
+
+on:
+  push:
+    branches:
+      - main
+      - fuzz-ci
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "core/src/**"
+      - "!core/src/docs/**"
+      - "!core/src/services/**"
+      - "core/src/services/fs/**"
+      - "core/src/services/memory/**"
+      - "core/src/services/s3/**"
+      - ".github/workflows/fuzz_test.yml"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+jobs:
+  fuzz-test:
```

Comment:
Maybe we need to use a more meaningful name, like ` fuzz-test-s3`?

Reply:
Please split them in different job so that they can run concurrently.

Reply:
solved by: https://github.com/apache/incubator-opendal/pull/2710/commits/7e497268765c9702893234fd6870cbaea70d92eb